LIMS_IMAGE/web/templates/home.html

```diff
@@ -1,14 +1,23 @@
-{% extends 'base.html' %} {% block title %}LIMS Home{% endblock %}
+{% extends 'base.html' %}
+{% load bootstrap5 %}
+{% load static %} 
+{% block title %}LIMS Home{% endblock %}
+<link rel="stylesheet" href="{% static 'css/home.css' %}"/>
 <body>
-  {% block content%}
-  <div class="container">
-    <h1>LIMS Home</h1>
-    <p>Welcome to the LIMS home page.</p>
-    {% if user.is_authenticated %} Hi {{ user.first_name }}!
-    <p><a href="{% url 'logout' %}">Log Out</a></p>
-    {% else %}
-    <p><i>You are not logged in</i></p>
-
-    {% endif %} {% endblock %}
-  </div>
+  {% block content %}
+    <div class="container p-4">
+      <h1>LIMS Home</h1>
+      {% if user.is_authenticated %} Hi {{ user.first_name }}!
+      <p><a href="{% url 'logout' %}">Log Out</a></p>
+      {% else %}
+      <p><i>You are not logged in</i></p>
+      <section id = "main">
+        <h2>Welcome to the LIMS home page</h2>
+        <div class ="container p-4">
+            <img src ="{% static 'image/images2.jpg' %}" alt ="Background image with Laboraroty theme" height = 500px width = 800px>
```

Comment:
don't set both height and width, this can lead to image stretching and looking odd. Set one or the other